simplelink/README

```diff
@@ -18,15 +18,22 @@ The driver library source is copied from the SDK, as follows:
   pushd simplelink_cc13x2_26x2_sdk_4_40_04_04/source/ti
```

Comment:
@Ayush1325 can you please update the version mentioned in the README also so that the details are in sync?